frontend/src/components/ViewTrips/save-trip-modal.js

```diff
@@ -249,18 +319,23 @@ class SaveTripModal extends React.Component {
         <Form>
           <Modal.Body>
             {createFormGroup('tripNameGroup', 'Trip Name', 'text',
-                    this.props.placeholderObj.name, this.nameRef)}
+                this.nameRef, this.isAddTripForm, 'Enter Trip Name',
+                this.props.defaultFormObj.name)}
             {createFormGroup('tripDescGroup', 'Trip Description', 'text',
-                    this.props.placeholderObj.description, this.descriptionRef)}
+                this.descriptionRef, this.isAddTripForm,
+                'Enter Trip Description', this.props.defaultFormObj.description)}
             {createFormGroup('tripDestGroup', 'Trip Destination', 'text',
-                    this.props.placeholderObj.destination, this.destinationRef)}
+                this.destinationRef, this.isAddTripForm,
+                'Enter Trip Destination', this.props.defaultFormObj.destination)}
             {createFormGroup('tripStartDateGroup', 'Start Date', 'date',
-                    this.props.placeholderObj.startDate, this.startDateRef)}
+                this.startDateRef, this.isAddTripForm, '',
+                this.props.defaultFormObj.startDate)}
             {createFormGroup('tripEndDateGroup', 'End Date', 'date',
-                    this.props.placeholderObj.endDate, this.endDateRef)}
+                this.endDateRef, this.isAddTripForm, '',
+                this.props.defaultFormObj.endDate)}
             {createFormGroup('tripCollabsGroup', 'Trip Collaborators', 'emails',
-                    this.props.placeholderObj.collaborators,
-                    this.state.collaboratorsRefArr)}
+                this.state.collaboratorsRefArr, this.isAddTripForm,
+                'person@email.xyz', this.props.defaultFormObj.collaborators)}
```

Comment:
Does the user need to append the @gmail.com here?

Reply:
Yes. They will need to add their whole email since we are planning to support multiple email domains.